ssn/integrated/informativeExamples/InkBird-IBS-TH2-instance.ttl

```diff
@@ -171,19 +180,19 @@
     rel:own <acmeBreweryCo> ;
     rel:own <acmeDelivery> ;
     sosa:observedProperty equipment:airTemperature ;
-    sosa:hasUltimateFeatureOfInterest <PO202402/BeerTemperature> ;
+    sosa:hasUltimateFeatureOfInterest <10/PO202402/21/0001/BeerTemperature> ;
```

Comment:
I would say the UltimateFeatureOfInterest is `<10/PO202402/21/0001/acmePorterSixPack>`, or even the beer in a specific bottle in the sixpack, since that is what the BeerTemperature is a property of. The BeerTemperature is an ObservedProperty: `<10/PO202402/21/0001/BeerTemperature> a sosa:ObservableProperty` so it can't be a Feature...?

Reply:
Fixed with Samples.